pkgs/servers/rpiplay/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, pkg-config, fetchFromGitHub, fetchpatch, cmake, wrapGAppsHook, avahi, avahi-compat, openssl, gst_all_1, libplist }:
+
+stdenv.mkDerivation rec {
+  pname = "rpiplay";
+  version = "2021-04-29";
+
+  src = fetchFromGitHub {
+    owner = "FD-";
+    repo = "RPiPlay";
+    rev = "4a65c7b84ff09acd7d6d494bb9037bbf106153a8";
+    sha256 = "06znqlj5b09z3mc965dyvmx2kh64fd7s571q34fh8r9ng98k351s";
+  };
+
+  patches = [
+    # allow rpiplay to be used with firewall enabled.
+    # sets static ports 7000 7100 (tcp) and 6000 6001 7011 (udp)
+    (fetchpatch {
+      name = "use-static-ports.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/FD-/RPiPlay/pull/196.patch";
+      sha256 = "08dy829gyhyzw2n54zn5m3176cmd24k5hij24vpww5bhbwkbabww";
+    })
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    openssl
+    libplist
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    avahi
+    avahi-compat
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+  ];
+
+  meta = with lib; {
+    inherit (src.meta) homepage;
```

Comment:
I would appreciate it we would hardcode the homepage to easily click on it.

Reply:
done